### DISCUSSION TITLE
Rule builder in v3.10.0

### DISCUSSION THREAD

**Discussion by ropese34** [nextgenhealthcare/connect#5465] [Q&A] [answered]
Hello, I am trying to add a filter at the Destination level using the rule builder. All of my #0-4 rules are enabled. No matter if I combined the filter criteria/conditions in one rule or separate into multiple like I have currently with "OR" operand, I still get syntax errors on validation despite using the rule builder tool/GUI exclusively.

**Comment by pacmano1** [comment, marked as answer]
Screenshot of rules?  And you need to show each rule detail.

**Reply by ropese34** [reply]
If it's expected I have to edit after (on in between) completing the rule build, it's not clear. 
![image](https://user-images.githubusercontent.com/108378567/195429758-21fd6bae-a046-4d04-870e-3f30e0ef50fd.png)

![image](https://user-images.githubusercontent.com/108378567/195429892-a9152cf5-1144-4123-8f58-b9396cebfcbe.png)

![image](https://user-images.githubusercontent.com/108378567/195430016-349f0a55-238e-45f3-aec4-3155f84c51bc.png)

![image](https://user-images.githubusercontent.com/108378567/195430119-64c9d1bb-19b5-44cf-a9d5-e9ceefbf95aa.png)

**Reply by pacmano1** [reply]
Your string values need to be in quotes.  

Before you do that look at the generated script tab, you will see something like:

![image](https://user-images.githubusercontent.com/44065187/195431527-fbb59fc3-364c-4d22-8949-e24cab5228e2.png)

**Comment by ropese34** [comment]
Right. Thanks. I thought it would do it for me. It did most it though :)

**Reply by pacmano1** [reply]
I would stop using that builder and just build an array of values you expect and call array methods to look for the values needed.
